server

```diff
@@ -63,8 +66,24 @@ end
 
 post '/boards/:board/cards' do
   board = boards[params[:board]]
+  card_id = SecureRandom.uuid
   redirect '/' unless board
 
-  cards << params.slice(:board, :column, :message).merge(username: cookies[:username])
+  card = params.slice(:board, :column, :message).merge(username: cookies[:username]).merge(votes: 0).merge(id: card_id)
+  card = JSON.parse(card.to_json, symbolize_names: true)
+  cards << card
```

Comment:
we don't have to push here we cant push in line 72 directly because we don't need the `card` variable